lib/internal/modules/cjs/loader.js

```diff
@@ -1309,6 +1310,16 @@ let resolvedArgv;
 let hasPausedEntry = false;
 /** @type {import('vm').Script} */
 
+const esModuleDescriptor = {
+  __esModule: {
```

Comment:
Maybe add a `null` prototype to this?
```suggestion
const esModuleDescriptor = {
  __proto__: null,
  __esModule: {
```

Reply:
What would it be needed for?